src/read.c

```diff
@@ -1694,6 +1696,11 @@ void ReadLiteral (
         Match( S_TILDE, "~", follow );
         break;
 
+    /* '?' */
+    case S_HELP:
+        ReadHelp(follow);
+        break;
+
```

Comment:
I don't think you want this case -- perhaps put a SyntaxError in, or just let the code fall through to an error.

This lets you write `2 + ?Group` and have help appear, which is (a) weird, and (b) makes GAP crash when you exit the help.

Reply:
I don't even know why I put the case into `ReadLiteral`, it doesn't make sense. At best `?...` should be a statement (-ish, since it's not terminated by a semicolon...).